tokio/Cargo.toml

```diff
@@ -123,6 +123,7 @@ optional = true
 [dev-dependencies]
 tokio-test = { version = "0.2.0" }
 futures = { version = "0.3.0", features = ["async-await"] }
+futures-test = "0.3.0"
```

Comment:
can we pull what you use in here into tokio-test?

Reply:
Only imports count_waker, since I didn't want to write it myself and it adds some value in the tests. Could be integrated into tokio too.